pandas/io/parsers/base_parser.py

```diff
@@ -129,7 +129,6 @@ def __init__(self, kwds) -> None:
 
         self.true_values = kwds.get("true_values")
         self.false_values = kwds.get("false_values")
-        self.mangle_dupe_cols = kwds.get("mangle_dupe_cols", True)
```

Comment:
I didn't - can you try refreshing please? I suspect GitHub is showing you the merge commit, because if I look at https://github.com/pandas-dev/pandas/pull/49024/files then I don't see this change

Reply:
Yes, you're right.